### PR TITLE
Update working-groups.md to add JuliaMolSim

### PR DESCRIPTION
**Pull request by ViralBShah**
cc @rkurchin

### CODE REVIEW

File: community/working-groups.md [hunk 1]
```diff
@@ -16,11 +16,12 @@ UTC refers to [Coordinated Universal Time](https://en.wikipedia.org/wiki/Coordin
 | Working Group                    | Slack channel       | Meetings                                 | Contact                                                                  |
 | -------------------------------- | ------------------- | ---------------------------------------- | ------------------------------------------------------------------------ |
 | Astronomy                        | `#astro`            | Fourth Friday of each month at 19:00 UTC | [Mosè Giordano][Mosè-Giordano], [Miles Lucas][Miles-Lucas]               |
-| CI infrastructure                | `#ci-dev`           | Every other Monday at 12pm ET             | [Elliot Saba][Elliot-Saba], [Dilum Aluthge][Dilum-Aluthge]               |
-| [Dagger][dagger]                 | `#distributed`      | Every other Thursday at 10am ET          | [Julian Samaroo][Julian-Samaroo], [Krystian Guliński][Krystian-Guliński]    |
+| CI infrastructure                | `#ci-dev`           | Every other Monday at 12pm ET             | [Elliot Saba][Elliot-Saba], [Dilum Aluthge][Dilum-Aluthge]              |
```
Comment (giordano) [inline community/working-groups.md:19]: This isn't happening anymore

File: community/working-groups.md [hunk 2]
```diff
@@ -16,11 +16,12 @@ UTC refers to [Coordinated Universal Time](https://en.wikipedia.org/wiki/Coordin
 | Working Group                    | Slack channel       | Meetings                                 | Contact                                                                  |
 | -------------------------------- | ------------------- | ---------------------------------------- | ------------------------------------------------------------------------ |
 | Astronomy                        | `#astro`            | Fourth Friday of each month at 19:00 UTC | [Mosè Giordano][Mosè-Giordano], [Miles Lucas][Miles-Lucas]               |
```
Comment (giordano) [inline community/working-groups.md:18]: @barrettp took over this one, but I'm not sure what's the new schedule of the meetings

File: community/working-groups.md [hunk 3]
```diff
@@ -16,11 +16,12 @@ UTC refers to [Coordinated Universal Time](https://en.wikipedia.org/wiki/Coordin
...
-| [Dagger][dagger]                 | `#distributed`      | Every other Thursday at 10am ET          | [Julian Samaroo][Julian-Samaroo], [Krystian Guliński][Krystian-Guliński]    |
+| CI infrastructure                | `#ci-dev`           | Every other Monday at 12pm ET             | [Elliot Saba][Elliot-Saba], [Dilum Aluthge][Dilum-Aluthge]              |
+| [Dagger][dagger]                 | `#distributed`      | Every other Thursday at 10am ET          | [Julian Samaroo][Julian-Samaroo], [Krystian Guliński][Krystian-Guliński] |
 | [Documentation tooling][wg-docs] | `#documentation`    | Last Tuesday of every month at 18:00 UTC | [Morten Piibeleht][Morten-Piibeleht], [Fredrik Ekre][Fredrik-Ekre]       |
 | [GPU][gpu]                       | `#gpu`              | Every other Monday at 11am ET            | [Tim Besard][Tim-Besard], [Valentin Churavy][Valentin-Churavy]           |
 | High-performance computing (HPC) | `#hpc`              | Fourth Tuesday of each month at 2pm ET   | [Valentin Churavy][Valentin-Churavy]                                     |
```
Comment (giordano) [inline community/working-groups.md:23]: Valentin and I are running another meeting the second Thursday of the month at 14:00 central Europe time

File: community/working-groups.md (hunk 2 above)
Comment (barrettp) [reply community/working-groups.md:18]: The Astronomy meetings are typically the fourth Thursday of each month except when they occur on a holiday such as Thanksgiving in the US or Xmas.

File: community/working-groups.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-## Working Groups
+v## Working Groups
```
Comment (giordano) [inline community/working-groups.md:1]:
```suggestion
## Working Groups
```

**Comment by ViralBShah** [comment]
I'd like to keep this PR to adding the JuliaMolSim meetup. Can we have a separate PR for all the updates discussed here?

**Comment by ViralBShah** [comment]
@rkurchin Is there a slack channel for juliamolsim? Not necessarily required, but if there is one, we should add it to the table before merging the PR.

File: community/working-groups.md (hunk 2 above)
Comment (ViralBShah) [reply community/working-groups.md:18]: @barrettp Can you submit a PR with the update?

File: community/working-groups.md (hunk 3 above)
Comment (ViralBShah) [reply community/working-groups.md:22]: I am not sure I understand. Best to make a PR. I deleted the CI one.

File: community/working-groups.md (hunk 1 above)
Comment (ViralBShah) [reply community/working-groups.md]: Deleted

File: community/working-groups.md (hunk 3 above)
Comment (giordano) [reply community/working-groups.md:22]: #2198

**Pull request merged**